.github/workflows/test-build-spack.yml

```diff
@@ -0,0 +1,94 @@
+name: Build ESMF Using Spack
+
+env:
+  esmf_version: 'esmf@develop'
+  nuopc_app_version: 'develop'
+
+on:
+  workflow_dispatch:
+    inputs:
+      esmf_version: 
+        description: 'ESMF version or tag like esmf@develop or esmf@=8.5.0b23'
+        required: false
+        type: string
+        default: 'esmf@develop'
+      nuopc_app_version:
+        description: 'NUOPC Application Prototypes version or tag like develop or v8.5.0b23'
+        required: false
+        type: string
+        default: 'develop'
+  schedule:
+    - cron: '0 6 * * *'
+    - cron: '0 18 * * *'
```

Comment:
@uturuncoglu - We are already past 18:00 UTC. Maybe use `45 18 * * *` to start in a few minutes from now.

Reply:
Sorry, I just reloaded the action status, and see the failed run from earlier... which ran at 18:00 UTC. Having to reload on that page always gets me.